apps/server/src/modules/files-storage/controller/files-storage.controller.ts

```diff
@@ -100,7 +100,7 @@ export class FilesStorageController {
 	@ApiResponse({ status: 406, type: NotAcceptableException })
 	@ApiResponse({ status: 500, type: InternalServerErrorException })
 	@ApiHeader({ name: 'Range', required: false })
-	@Get('/download/:fileRecordId/:fileName')
+	@Get('/download/:fileRecordId')
```

Comment:
Why do you remove fileName ?

Reply:
There is no technical necessity to use the file name in the api. It was only used for validation of the file record. When we use the file service for external tools then we will only have the file record id as a reference. Otherwise we would have to save the file name too.

Reply:
The links should be readable for humans